Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -315,6 +325,12 @@ RDKit::SparseIntVect<boost::uint32_t> *MorganFingerprintHelper(
     python::object fromAtoms, bool useChirality, bool useBondTypes,
     bool useFeatures, bool useCounts, python::object bitInfo) {
   std::vector<boost::uint32_t> *invars = 0;
+  if (radius < 0) {
```

Comment:
For radius a better solution would be to change the function signature from "int radius" to "unsigned int radius", the boost wrapper then takes care of the negative numbers.

Reply:
Also change nbits to "unsigned int nBits" would be appropriate here as well.